Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -365,12 +408,22 @@ namespace O3DE::ProjectManager
 
     void CreateGem::HandleBackButton()
     {
-        if (m_tabWidget->currentIndex() > 0)
+        if (m_stackWidget->currentIndex() > gemTemplateSelectionScreen)
         {
-            m_tabWidget->setCurrentIndex(m_tabWidget->currentIndex() - 1);
+            m_stackWidget->setCurrentIndex(m_stackWidget->currentIndex() - 1);
+
+            int newIndex = m_stackWidget->currentIndex();
+            if (newIndex == gemDetailsScreen)
+            {
+                m_gemDetailsTab->setChecked(true);
+            }
+            else if (newIndex == gemTemplateSelectionScreen)
+            {
+                m_gemTemplateSelectionTab->setChecked(true);
+            }
```

Comment:
might be slightly more readable like this: 
```suggestion
            const newIndex = m_stackWidget->currentIndex() - 1;
            m_stackWidget->setCurrentIndex(newIndex );
            
            if (newIndex == gemDetailsScreen)
            {
                m_gemDetailsTab->setChecked(true);
            }
            else if (newIndex == gemTemplateSelectionScreen)
            {
                m_gemTemplateSelectionTab->setChecked(true);
            }
```
Is  there a missing case - aren't there 3 buttons?

Reply:
Alternatively, we could add the radio buttons as they are created to an exclusive button group. This could then just get the button for the current page index and setChecked on that. Then we wouldn't need to remember to update this if a new tab is added/removed.